diagnostic.md

```diff
@@ -59,6 +66,8 @@ code to import the mixin definition in the current module.
 
 ```scss
 // your response here
+@import './row.scss';
+
 ```
```

Comment:
This would *technically* work, but it's not how we'd want to import modules in SASS. A better approach is

```CSS
@import 'row';
```

just like we did in class yesterday.